discord/client.py

```diff
@@ -721,13 +721,16 @@ def stop_loop_on_completion(f):
     # properties
 
     def is_closed(self) -> bool:
-        """:class:`bool`: Indicates if the WebSocket connection is closed."""
+        """Indicates if the WebSocket connection is closed."""
         return self._closed
 
     @property
     def activity(self) -> ActivityTypes | None:
-        """Optional[:class:`.BaseActivity`]: The activity being used upon
-        logging in.
+        """The activity being used upon logging in.
+
+        Returns
+        -------
+        Optional[:class:`.BaseActivity`]
```

Comment:
Why is this out of line with the actual return type annotation of this property? Is there a good reason, and if so, could it be fixed in another PR?

Reply:
It's just keeping the original type, maybe could be changed in the future